crates/indexer/src/models/signatures.rs

```diff
@@ -81,16 +80,16 @@ impl Signature {
                 transaction_version,
                 transaction_block_height,
             ),
-            APITransactionSignature::Secp256k1EcdsaSignature(sig) => {
-                Ok(vec![Self::parse_secp256k1_ecdsa_signature(
+            APITransactionSignature::SingleSender(sig) => {
+                Ok(Self::parse_multi_agent_signature_helper(
```

Comment:
Was this supposed to be `multi_agent` or `multi_key`?

Reply:
hack for legacy indexer